server/src/index.ts

```diff
@@ -404,6 +414,16 @@ app.get('*', (_, res) => {
   );
 });
 
+app.post('/webhooks', (req, res) => {
+  const isWebhookVerified = verifyWebhookSecret(req);
+  if (!isWebhookVerified) {
+    res.status(400).send('Request not verified');
```

Comment:
let's use the debug package here so if some people have an issue we can see what's going on

Reply:
Sounds good, will do.